demos/workbench/workbench.py

```diff
@@ -545,22 +547,16 @@ def update_llm_model_options(selected_llm_provider, llm_model):
     pdf_list.select(
         fn=handle_pdf_selection,
         inputs=[pdf_list],
-        outputs=[show_img, chatbot, feedback_markdown, pdf_selector_modal],
+        outputs=[show_pdf, chatbot, feedback_markdown, pdf_selector_modal]
     )
 
     # First close the modal when user selects a file
     upload_btn.click(fn=lambda: gr.update(visible=False), outputs=pdf_selector_modal)
 
     upload_btn.upload(
         fn=render_first,
-        inputs=[
-            upload_btn,
-            chunk_size,
-            chunking_technique,
-            selected_embedding_model,
-            session_state,
-        ],
-        outputs=[show_img, chatbot, session_state],
+        inputs=[upload_btn, chunk_size, chunking_technique, session_state],
+        outputs=[show_pdf, chatbot, session_state],
```

Comment:
@bsbodden correct me if wrong, but I think we need to make sure the selected embedding model is passed through here. once we update that we should be able to merge